src/Microsoft.DotNet.Interactive.Formatting/HtmlFormatter.cs

```diff
@@ -226,22 +226,7 @@ void Initialize()
             var formatter = GetDefaultFormatterForAnyObject(type);
             return formatter.Format(value, context);
         }),
-
-        // Final last resort is to convert to plain text
-        new HtmlFormatter<object>((value, context) =>
```

Comment:
Was this removed because it was redundant?

Reply:
Yes. There were two different registrations for the same type. This one had no test coverage so it wasn't important.